docs/ide4c/vscode/README.md

```diff
@@ -0,0 +1,7 @@
+# Configuring VS Code for ICU4C
+
+  - create `.vscode` folder in icu4c/source
+  - Copy `tasks.json` and `launch.json` files into `.vscode` folder
+
+NOTE
+ before you build `icu4c` from vs code. run `./runConfigureICU` command first.
```

Comment:
Please specify whether this has to be an in-source build, or else how to tell VS Code about the relationship between the source and output folders.

Reply:
Done ... I added a link to the page that talks about build/test ICU. There is no place in master/docs that explain the building process yet.